bmi_calculator/lib/calculator_page.dart

```diff
@@ -0,0 +1,400 @@
+import 'package:bmi_calculator/result_page.dart';
```

Comment:
This file is unreadable as it is a giant nested Page. Break the nested widgets into their own properties or functions, so your final Calculator page looks more like

```
Calculator page
   ScaffoldWidget
     BodyWidget
       Content
         Gender Selector Widget
         Counter Widget
         Weight Widget
         Height Widget
```
where each widget is declared separately i.e Gender Selector Widget is encapsulated in a function or rather written in a new file as a Stateful Widget.